### PR TITLE
UF-9373: Add municipalityId and update MessagingIntegration

### PR DESCRIPTION
**Pull request by cheezi0747**

### CODE REVIEW

File: mockfile.xls
Comment (foobar-1337) [inline mockfile.xls:1]: What is this?

File: src/main/java/se/sundsvall/citizenchanges/integration/messaging/MessagingClient.java [hunk 1]
```diff
@@ -23,16 +24,20 @@ public interface MessagingClient {
...
-	MessageResult sendEmail(EmailRequest request);
+	@PostMapping("/{municipalityId}/email")
+	MessageResult sendEmail(
+		@PathVariable("municipalityId") String municipalityId,
+		EmailRequest request);
 
 	/**
 	 * Send a single sms
 	 *
 	 * @param request containing sms information
```
Comment (foobar-1337) [inline src/main/java/se/sundsvall/citizenchanges/integration/messaging/MessagingClient.java:35]: Update javadoc according to new method signature

File: src/main/java/se/sundsvall/citizenchanges/integration/messaging/MessagingClient.java [hunk 2]
```diff
@@ -23,16 +24,20 @@ public interface MessagingClient {
 	 * @param request containing email information
 	 * @return response containing id and delivery results for sent message
```
Comment (foobar-1337) [inline src/main/java/se/sundsvall/citizenchanges/integration/messaging/MessagingClient.java:25]: Update javadoc according to new method signature

File: src/test/java/se/sundsvall/citizenchanges/integration/citizen/CitizenIntegrationTests.java [hunk 3]
```diff
@@ -62,19 +62,19 @@ void getAddressChanges_ok() {
...
-		assertThat(a.getCo()).isEqualTo("someCo");
-		assertThat(a.getAppartmentNumber()).isEqualTo("someApartmentNumber");
-		assertThat(a.getPostalCode()).isEqualTo("somePostalCode");
-		assertThat(a.getCity()).isEqualTo("some");
-		assertThat(a.getCounty()).isEqualTo("81");
-		assertThat(a.getMunicipality()).isEqualTo("22");
-		assertThat(a.getAddressType()).isEqualTo("someAddressType");
-		assertThat(a.getxCoordLocal()).isEqualTo(12D);
-		assertThat(a.getyCoordLocal()).isEqualTo(14D);
+	private void assertAddressValues(final CitizenAddress adress) {
```
Comment (foobar-1337) [inline src/test/java/se/sundsvall/citizenchanges/integration/citizen/CitizenIntegrationTests.java:65]: typo (adress-> address)

File: mockfile.xls
Comment (cheezi0747) [reply mockfile.xls:1]: a excel file with a single row used in tests.

**Review by AndreSoderlund — DISMISSED** [review]

**Pull request merged**